metric/metricexport/producer.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 // Producer is a source of metrics.
+// Deprecated in lieu of go.opencensus.io.metric.producer.Producer
```

Comment:
Simply delete it, we don't use this anywhere correct?

Reply:
deleted.